src/wrapping.rs

```diff
@@ -90,6 +159,24 @@ impl<T: Serialize> Serialize for Wrapping<T> {
     }
 }
 
+#[cfg(all(test, feature = "num-traits"))]
+mod tests {
+    use num_traits::{Num, NumOps, Zero, One};
+    use core::ops::{Div, Rem, Mul, Add, Sub};
+    use crate::{Integer, NonZero};
+
+    use crate::{Wrapping, U64};
+
+    fn assures_num<T: Num>(x: T) {}
+
+    #[test]
+    fn num_traits() {
+        const TEST: Wrapping<U64> = Wrapping(U64::from_u64(0x0011223344556677));
+
+        assures_num(TEST)
```

Comment:
My logic here is: `Num` is implemented for `Wrapping<T> where ...` so that we're implementing it for the minimally restricted type, but we want to assure that it works for `Wrapping<Uint<Limbs>>`, so this code assures it.

I'm not sure if there's a better way of checking this.